keyberon/src/layout.rs

```diff
@@ -1975,14 +1979,20 @@ impl<'a, const C: usize, const R: usize, T: 'a + Copy + std::fmt::Debug> Layout<
 
     /// Returns a list indices of layers that should be used for [`Action::Trans`] resolution.
     pub fn trans_resolution_layer_order(&self) -> LayerStack {
```

Comment:
Hm reading this again, "Stack" isn't really the right name is it? It's first-in first-out so is a queue.

Reply:
Yea, but the "outside" concept is that the new trans keys behavior is stack-like. 

I would also argue that it isn't queue either, since there's no actual queueing up at any stage. Only loading all layers at once into memory. It's done reverse order which is unlike how usual stack memory layout, and like queue. But that's about where similarities/differences end.

Reply:
Added explanation in [`a4ede26` (#948)](https://github.com/jtroo/kanata/pull/948/commits/a4ede2623ecb1aac9fd65447ef8bc240ecd4db38). Is this enough now?